db/migrate/20190129185647_create_finding_aid_model.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CreateFindingAidModel < ActiveRecord::Migration[5.2]
+  def change
+    create_table :finding_aids do |t|
+      t.string :name
+      t.text :description
+      t.string :subject
+      t.string :content_link
+      t.string :identifier
+      t.references :collection, foreign_key: true
+
+      t.timestamps
+    end
+    create_table :finding_aid_responsibilities do |t|
```

Comment:
This is plural `finding_aid_responsibilities` while the schema still has the singular `finding_aid_responsibility`

Reply:
The schema was wrong, it should be pluralized.